src/stats-template.html

```diff
@@ -41,29 +39,35 @@ <h1 tabindex="-1">My Assessment</h1>
   <ul aria-label="menu">
     <li>
       {{#if incomplete}}
-      <button class="nav-link pick" data-action="start-or-resume">Practise some questions</button>
+      <button class="nav-link pick" data-action="start-or-resume">Try some questions</button>
       {{else}}
-      <button class="nav-link pick" data-action="restart">Practise again</button>
+      <button class="nav-link pick" data-action="restart">Try again</button>
       {{/if}}
     </li>
     <li>
-      <button class="nav-link break" data-action="break">I&#x2019;ll have a break</button>
+      <button class="nav-link break" data-action="break">Have a break</button>
     </li>
     <li>
-      <button class="nav-link break" data-action="menu">##MISSING COPY##Take me to information##MISSING COPY##</button>
+      <button class="nav-link break" data-action="menu">Read more about PIP</button>
     </li>
   </ul>
 </div>
 <div class="box no-print" id="data-box">
-  <h2>My Data</h2>
-  <p>This app does not send any of your data anywhere or keep any for itself. However, it <em>does</em> save some data just on the computer you&#x2019;re using in order to work properly.</p>
-  <p>If this is not your computer, tablet or phone you can delete all this data when you are done using the app. This way, no one else will see it.</p>
+  <h2>Your Data</h2>
+  <p>This site does not send any of your data anywhere. It does collect some anonymous information such as how many people have used the tool so we can make improvements.</p>
+  <p>However, it does save some data just on the computer you’re using in order to work properly. If this is not your computer, tablet or phone you can delete all this data when you are done using the site. This way, no one else will see it.</p>
```

Comment:
Same comment on ESA. Saying It does... in sentence 2 then However in sentence 3 doesn't make much sense.